components/mfa/confirm.tsx

```diff
@@ -41,6 +43,7 @@ export default class Confirm extends React.PureComponent<Props> {
 
     submit = (e: KeyboardEvent | React.FormEvent<HTMLFormElement>): void => {
         e.preventDefault();
+        trackEvent('mfa_setup', 'mfa_confirmed', {isMobile: isMobile()});
```

Comment:
Why are we only tracking mobile or desktop? Maybe we should make this more generic, so we can have data about the desktop app (which shares the same codebase) and the mobile app (eventually).

Reply:
I'll reuse the utils function from this PR https://github.com/mattermost/mattermost-webapp/pull/12029, thanks!

Reply:
A minor comment is that current event name convention is *Action/Event Taken* + _ + *Object Taken Action On (optional - if any)* + _ + *Purpose/Result Of The Action*.

Perhaps `mfa_confirmed` can be renamed to `submit_mfa_confirm`.

Reply:
Done, changed to `submit_mfa_confirm`